datasets/ONS-UK-trade-in-goods-by-industry-country-and-commodity/concat_main.py

```diff
@@ -0,0 +1,118 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.6.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# UK trade in goods by industry, country and commodity, exports and imports
+
+import pandas as pd
+import json
+from gssutils import *
+
+pd.options.mode.chained_assignment = None 
+
+cubes = Cubes("info.json")
+
+# +
+with open ('info.json') as file:
+    info = json.load(file)
```

Comment:
Can shorten this to: `info = json.load(open('info.json'))`
Not a huge necessity but can be neat to have it on a single line.